src/main/resources/static/css/custom.css

```diff
@@ -30,3 +30,7 @@ main {
     -moz-font-feature-settings: 'liga';
     -moz-osx-font-smoothing: grayscale;
 }
+
+.pagination li.active {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (li.active) is overqualified, just use .active without element name.](https://www.codacy.com/app/matikz1110/PetFinder/file/2642351515/issues/source?bid=3239464&fileBranchId=3321823#l34)